translations/nm-tray_de.ts

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="de">
+<context>
+    <name>ConnectionInfo</name>
+    <message>
+        <location filename="../src/connectioninfo.ui" line="14"/>
+        <source>Connection information</source>
+        <translation>Verbindungsinformationen</translation>
+    </message>
+</context>
+<context>
+    <name>NmList</name>
+    <message>
+        <location filename="../src/nmlist.ui" line="14"/>
+        <source>Dialog</source>
+        <translatorcomment>Did not translate as &quot;Dialog&quot; seems to get replaced with string &quot;nm-tray info&quot; from context &quot;Tray&quot;.</translatorcomment>
+        <translation>Dialog</translation>
+    </message>
+    <message>
+        <location filename="../src/nmlist.ui" line="20"/>
+        <source>All information</source>
+        <translation>Verfügbare Informationen</translation>
+    </message>
+    <message>
+        <location filename="../src/nmlist.ui" line="30"/>
+        <source>Active connections</source>
+        <translation>Aktive Verbindungen</translation>
+    </message>
+    <message>
+        <location filename="../src/nmlist.ui" line="40"/>
+        <source>Available wireless</source>
+        <translation>Verfügbare drahtlose Verbindungen</translation>
+    </message>
+</context>
+<context>
+    <name>NmModel</name>
+    <message>
+        <location filename="../src/nmmodel.cpp" line="745"/>
+        <source>root</source>
+        <translatorcomment>Translated as &quot;network&quot; hence not literally at all. Not sure what&apos;s best here but this seems a bit more reasonable to me.</translatorcomment>
```

Comment:
Here applies the same as before.... this particular node will not be presented to user except the debug information.